File: eRecipes/UI/erecipes_desktop/lib/models/katalog_recept.dart
import 'package:erecipes_desktop/models/katalog.dart';
import 'package:erecipes_desktop/models/recept.dart';
import 'package:json_annotation/json_annotation.dart';

part 'katalog_recept.g.dart';

@JsonSerializable()
class KatalogRecept {
  int? katalogReceptId;
  int? receptId;
  int? katalogId;
  Recept? recept;
  Katalog? katalog;

  KatalogRecept();

  factory KatalogRecept.FromJson(Map<String, dynamic> json) =>
      _$KatalogReceptFromJson(json);

  Map<String, dynamic> toJson() => _$KatalogReceptToJson(this);
}

File: eRecipes/eRecipes.Model/KatalogRecept.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eRecipes.Model
{
    public class KatalogRecept
    {
        public int KatalogReceptId { get; set; }
        public int KatalogId { get; set; }
        public virtual Katalog? Katalog { get; set; }
        public int ReceptId { get; set; }
        public virtual Recept? Recept { get; set; }
    }
}
